### ISSUE TITLE
NoSuchMethodError

### ISSUE THREAD

**Issue by vimalaguti** [shadaj/scalapy#33]
Hi, for some reason hard to debug, I cannot initialize scalapy library.

I use this line to start scala: 
`JAVA_OPTS="-Djava.library.path=/home/abcde/.local/lib/python3.6/site-packages/jep" LD_PRELOAD=/usr/lib/python3.6/config-3.6m-x86_64-linux-gnu/libpython3.6.so:$LD_PRELOAD  amm`

then import scalapy:

`import $ivy.`me.shadaj::scalapy-core:0.3.0`, me.shadaj.scalapy.py`

And I get error:

```scala
val listLengthPython = py.global.len(List(1, 2, 3)) 
jep.JepException: <class 'RuntimeError'>: getPyObject
  jep.Jep.set(Native Method)
  jep.Jep.set(Jep.java:762)
  jep.Jep.setupJavaImportHook(Jep.java:297)
  jep.Jep.configureInterpreter(Jep.java:285)
  jep.Jep.<init>(Jep.java:256)
  jep.Jep.<init>(Jep.java:228)
  me.shadaj.scalapy.py.JepInterpreter.<init>(JepInterpreter.scala:9)
  me.shadaj.scalapy.py.Platform$.newInterpreter(Platform.scala:5)
  me.shadaj.scalapy.py.package$.interpreter(package.scala:11)
  me.shadaj.scalapy.py.Writer$$anon$11.write(Writer.scala:75)
  me.shadaj.scalapy.py.Any$.from(Any.scala:28)
  ammonite.$sess.cmd1$.<init>(cmd1.sc:1)
  ammonite.$sess.cmd1$.<clinit>(cmd1.sc)
java.lang.NoSuchMethodError: getPyObject
  jep.Jep.set(Native Method)
  jep.Jep.set(Jep.java:762)
  jep.Jep.setupJavaImportHook(Jep.java:297)
  jep.Jep.configureInterpreter(Jep.java:285)
  jep.Jep.<init>(Jep.java:256)
  jep.Jep.<init>(Jep.java:228)
  me.shadaj.scalapy.py.JepInterpreter.<init>(JepInterpreter.scala:9)
  me.shadaj.scalapy.py.Platform$.newInterpreter(Platform.scala:5)
  me.shadaj.scalapy.py.package$.interpreter(package.scala:11)
  me.shadaj.scalapy.py.Writer$$anon$11.write(Writer.scala:75)
  me.shadaj.scalapy.py.Any$.from(Any.scala:28)
  ammonite.$sess.cmd1$.<init>(cmd1.sc:1)
  ammonite.$sess.cmd1$.<clinit>(cmd1.sc)

```

Than I have to quit ammonite and reopen, because jep refuses to open a new instance.
I cannot even compile the example because its missing the jep implicit. (should I provide one?)
The same happens when I just try to import a module

**Comment by bsteffensmeier** [comment]
A getPyObject method was added in jep 3.9 for internal use. I suspect you are using jep 3.9 native library in you Python installation that is expecting that method to exist. But if you jep.jar is still from 3.8.x that method will not exist and you will see the error. Since we released jep 3.9 it will be the default version installed by pip. You may need to upgrade your jar to 3.9 or specify an older version when installing jep into your python.

**Comment by shadaj** [comment]
This is fixed on master, since we have moved to our own implementation for interfacing with CPython on the JVM.
